### ISSUE TITLE
[Feature Request] A short description of what you want to happen

### ISSUE THREAD

**Issue by hrst** [barbushin/php-imap#646]
Hi,

feature idea: if you add a `)` here:

https://github.com/barbushin/php-imap/blob/f401d85c8a6b034e8ee566e73cb948dd9e9298be/src/PhpImap/IncomingMail.php#L212

it will also catch background images!  ( `background-image: url(...)` )

Full new line:
`\preg_match_all("/\bcid:[^'\"\s)]{1,256}/mi", $fetchedHtml, $matches);`

**Comment by Sebi94nbg** [comment]
Do you mean like this? https://github.com/barbushin/php-imap/blob/develop/src/PhpImap/IncomingMail.php#L204-L246 :)

**Comment by hrst** [comment]
> Do you mean like this? https://github.com/barbushin/php-imap/blob/develop/src/PhpImap/IncomingMail.php#L204-L246 :)

Yes but with the above change it will also work for CSS background images which under my impression it is currently not.
